File: web3rpc/rpc-specs/code-samples/javascript/admin/getSpamThrottlerCandidateList.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    provider.admin.getSpamThrottlerCandidateList({}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: web3rpc/rpc-specs/code-samples/python/admin/getSpamThrottlerCandidateList.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

w3 = Web3(Web3.HTTPProvider(host))
admin_response = w3.geth.admin.get_spam_throttler_candidate_list()

print(admin_response)
